initfiles/componentfiles/configxml/esp.xsl

```diff
@@ -166,6 +159,17 @@
                     </verify>
                 </EspProtocol>
             </xsl:if>
+            <xsl:if test="string($controlPort) != ''">
```

Comment:
I will add the code to check both  controlPort == 0 and  controlPort == "".

It does default to "ON" and 8010 (please see the line 710 of initfiles/componentfiles/configxml/esp.xsd.in).